spec/features/current_event_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+feature "A user only sees information for the currently selected event" do
+  let!(:normal_user) { create(:user) }
+
+  scenario "User flow when there is a live event" do
+    event_1 = create(:event, state: 'open')
```

Comment:
Nothing wrong with setting state like this, but you could also do a factory girl trait :open if this type of thing would be used frequently.